content/millennium/dstu2/medications/medication-statement.md

```diff
@@ -65,21 +71,16 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2
 
  ID                              | Value\[x] Type      | Description
 ---------------------------------|---------------------|-------------------------------------------------------------------------------------------
- `patient-friendly-display`      | [`string`]          | The display that can be used for this field when producing a view suitable for a patient.
- `medication-statement-category` | [`CodeableConcept`] | The [category] of the order, for example: patientspecified, outpatient, etc.
+ `patient-friendly-display`      | [`string`]          | Extension to MedicationRequest.dosageInstruction. The display name that can be used for this field when producing a view suitable for a patient.
+ `medication-statement-category` | [`CodeableConcept`] | The [category] of the order, for example: patientspecified, outpatient, and so on.
```

Comment:
having "and so on" here is redundant since it already says they are examples of values, not the complete set